README.md

```diff
@@ -1,7 +1,42 @@
-# linters-config
+# # Positioning and Floating Elements\
 
-## style command 
-npx stylelint "**/*.{css,scss}"
+> created a clone of a ny times article
 
-## html command 
-npx hint .
\ No newline at end of file
+![screenshot](/assets/img/pagescreenshot.png)
+
+## Built With
+
+- html5
+- Css3
+
+## Live Demo
+
+https://rawcdn.githack.com/MkrtichSargsyan/microverse-ny-times-clone/ccb33f62add2d031236ad33d5d116cfc7a5798bf/index.html
```

Comment:
fix the live demo link it's looking ugly this way. use the same markdown used in the Github and Linkedin profile links.